docs/io/examples.rst

```diff
@@ -225,6 +225,9 @@ Read
     import pandas as pd
     from kartothek.io.dask.delayed import read_dataset_as_delayed
 
-    tasks = read_dataset_as_delayed(dataset_uuid="MyFirstDatasetIter", store=store_factory)
+    tasks = read_dataset_as_delayed(
+        dataset_uuid="MyFirstDatasetDask",
+        store=store_factory
+        )
```

Comment:
This will prob cause travis to fail due to improper formatting. consider installing the pre commit hooks